kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/DefaultExpressionUtils.java

```diff
@@ -383,4 +383,17 @@ private ColumnVector getVector(int rowId) {
       }
     };
   }
+  /** Escapes characters escapeChar in the input String */
```

Comment:
I think there is a similar method in `LikeExpressionEvaluator`. Move it to here.

Reply:
Done, moved this utils here.  

(LikeExpressionEvaluator's method is more like generating regex for evaluating the rows)
https://github.com/delta-io/delta/blob/4dbadbbf8ddd0a12273ac9521d61bc89196dc80d/kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/LikeExpressionEvaluator.java#L159